CHANGES

```diff
@@ -11138,7 +11138,7 @@ Version 1.3.26 (October 9, 2005)
               valuewrapper
 
             It is now possible, for example to ignore all methods/classes in a header file, except for a
-            few targetted methods, for example:
+            few targeted methods, for example:
```

Comment:
That's not a typo - "targetted" is the UK English spelling.

Reply:
Will update to `codespell` dictionary with this feedback. thanks. I'll revert this change.